projects/observability/src/shared/components/cartesian/cartesian-chart.component.ts

```diff
@@ -61,9 +62,20 @@ export class CartesianChartComponent<TData> implements OnChanges, OnDestroy {
   @Input()
   public selectedInterval?: IntervalValue;
 
+  @Input()
+  public sync?: boolean;
```

Comment:
API wise, I think this change makes sense with one suggestion - removing the boolean flag. It doesn't make sense to mark this true and not put a groupId, and likewise a groupId doesn't make sense if this isn't true, so the two properties are redundant - we can infer the sync behavior by the presence of a group ID.